pages/tidal tools/sourcecode.md

```diff
@@ -1,120 +1,107 @@
+## Tidal Source Code Analysis
 
-Not sure how ready you are to move to the cloud? With Tidal you have
-the option to analyze your specified source code associated with the applications.
-The analysis will identify the difficulty to migrate your applications to the
-cloud.
-
-If you are interested in a deeper dive source code assessment, let us know at
-[info@tidalcloud.com](mailto:info@tidalcloud.com).
-
-Try it out!
+Not sure how ready you are to move to the cloud? With Tidal you can analyze your
+application's source code to identify difficulties and opportunities in migrating and modernizing.
+It is compatible with most of the current and legacy languages and detects a wide array of factors. See [the overview
+for more information on its capabilities](/code-analysis-overview.html).
 
 {% include image.html file="source_code_analyze.png" caption="Analyze your source code" %}
 
-## Getting Started
+## What You Need to Get Started
 
-1. Install, configure and authenticate via Tidal Tools. Make sure you follow these guides.
+1. [Install](tidal-tools.html#install), and [authenticate](tidal-tools.html#connecting-to-the-api) Tidal Tools with your [Tidal Accelerator](tidalcloud.com/accelerator) Workspace.
 
-- How to [install](tidal-tools.html#install) Tidal Tools.
-- Install Tidal Tools [dependencies](tidal-tools.html#dependencies).
-- Make sure you can [connect](tidal-tools.html#connecting-to-the-api) with your workspace using Tidal Tools.
-- As the last step, You should run the [tidal doctor](troubleshooting.html#tidal-doctor) command to verify that your environment has been configured properly.
+2. You will need the ID of the application you are going to perform the source code analysis on. You can find it in the URL bar when looking at an application in Accelerator. ex. If you are viewing an application in Tidal, the URL will show `https://[your workspace].tidal.cloud/apps/111/overview` in this case 111 is the application ID.
 
-2. You will need the ID of the application for which you are going to perform the source code analysis. You can find it in the URL bar when looking at an application. ex. If you are viewing an application in Tidal, the URL will show `https://[your workspace].tidal.cloud/apps/111/overview` in this case 111 is the application ID.
+3. You will need a copy of the source code files for the application locally on your device.
 
-3. Lastly, you will need a local copy of the source code for the application.
+With that, you can perform the code analysis in a matter of seconds.
 
-4. Perform the Source Code Analysis. There are 2 options to do so. You can find more details in this [section](#perform-the-analysis).
+{% include tip.html content="Looking to try it out and don't have any code handy? You can use this sample ['schoolbus' application](https://github.com/tidalmigrations/schoolbus) by [cloning it from GitHub](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository)." %}
```

Comment:
Thought: We could swap out the readme in schoolbus and have it explain how to use the github code action, then give them the option of use the action as the quickest way of getting running

Reply:
interesting idea! maybe something to consider if in the future we need more usage/guidance on using it.